src/util/pg.py

```diff
@@ -112,6 +112,18 @@ def execute(query):
             )
 
 
+def format(cr, query, **kwargs):
+    """
+    Format the query replacing all keyword args as SQL indentifiers
+    Example:
+    ```
+    >>> util.format(cr, "SELECT id FROM {table}", table="res_users")
+    SELECT id FROM res_users
```

Comment:
Just to not forget updating the docstring :)
```suggestion
    SELECT id FROM "res_users"
```